data/sql/updates/pending_db_world/wip_ahnkahet_azc.sql

```diff
@@ -0,0 +1,90 @@
+-- Change this back to INSERT once rewrite is complete
+REPLACE INTO `version_db_world` (`sql_rev`) VALUES ('1599510903096685100');
+
+-- Amanitar encounter
+DELETE FROM `spell_script_names` WHERE `spell_id`=57283 AND `ScriptName`='spell_amanitar_remove_mushroom_power';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(57283, 'spell_amanitar_remove_mushroom_power');
+DELETE FROM `spell_script_names` WHERE `spell_id`=56648;
+DELETE FROM `spell_group_stack_rules` WHERE `group_id`=1113;
+INSERT INTO `spell_group_stack_rules` (`group_id`, `stack_rule`, `description`) VALUES
+(1113, 8, 'Ahn\'kahet - Potent Fogus and Mini');
+DELETE FROM `spell_group` WHERE `id`=1113;
+INSERT INTO `spell_group` (`id`, `spell_id`, `special_flag`) VALUES
+(1113, 57055, 0),
+(1113, 56648, 0);
+
+-- Teldaram
+DELETE FROM `spell_script_names` WHERE `spell_id` = 55931 AND `ScriptName`='spell_prince_taldaram_conjure_flame_sphere';
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (55895, 59511, 59512) AND `ScriptName` = 'spell_prince_taldaram_flame_sphere_summon';
+INSERT INTO `spell_script_names` (`spell_id` ,`ScriptName`) VALUES
+(55931, 'spell_prince_taldaram_conjure_flame_sphere'),
+(55895, 'spell_prince_taldaram_flame_sphere_summon'),
+(59511, 'spell_prince_taldaram_flame_sphere_summon'),
+(59512, 'spell_prince_taldaram_flame_sphere_summon');
+UPDATE `creature_text` SET `TextRange`='3',`comment`='prince taldaram SAY_SPHERE_ACTIVATED' WHERE  `CreatureID`=29308 AND `GroupID`=0 AND `ID`=0;
+UPDATE `creature_text` SET `TextRange`='3',`comment`='prince taldaram SAY_REMOVE_PRISON'  WHERE  `CreatureID`=29308 AND `GroupID`=1 AND `ID`=0;
```

Comment:
Don't use strings where they are not necessary :)

Reply:
Such comments are useful for development mostly, it allows to understand where texts are being to be used but indeed its not necessary. Anyways i hope this won't go against code of conduct but would like to use it, just it helps to read which and where texts are used for me personally (sure i can make sql comments but...)

Reply:
No I mean the `text_range` use string for the int

Reply:
which is not neccessary